### PR TITLE
Cancel order

### PR DESCRIPTION
**Pull request by rocha7778**

### CODE REVIEW

File: service/mantle/shopify/ApiServices.xml [hunk 1]
```diff
@@ -539,17 +550,7 @@
...
-            </entity-find-one>
-
-            <set field="productStoreId" from ="systemRemoteIdentification.productStoreId"/>
-
-            <entity-find-one entity-name="mantle.product.store.ProductStoreSetting" value-field="productLocation">
-                <field-map field-name="productStoreId" from="productStoreId"/>
-                <field-map field-name="settingTypeEnumId" value="ShopifyStoreId"/>
-            </entity-find-one>
-            <set field="locationId" from="productLocation.settingValue"/>
+            <service-call name="mantle.shopify.ShopifyServices.get#LocationId" out-map="context"   in-map="[ remoteId:'DEMO_SHOPIFY_SET_INVENTORY_REMOTE']"/>
```
Comment (marror7) [inline service/mantle/shopify/ApiServices.xml:553]: Remove blank space on in-map

File: service/mantle/shopify/ApiServices.xml [hunk 2]
```diff
@@ -587,4 +588,159 @@
...
+
+            <set field="externalId" from="orderHeader.externalId" />
+            <set field="currencyUomId" from="orderHeader.currencyUomId" />
+            <set field="shipping.full_refund" value="true"/>
+            <set field="refund.shipping" from="shipping"/>
+            <set field="refund.currency" from="currencyUomId"/>
+            <set field="refund.refund_line_items" from="refund_line_items"/>
+            <set field="refund.orderStatus" from="orderHeader.statusId" />
+            <set field="refund.orderId" from="externalId" />
+            <set field="messageText" from="groovy.json.JsonOutput.toJson([
```
Comment (marror7) [inline service/mantle/shopify/ApiServices.xml:636]: Line 636, 637, 638 can be in one.

File: service/mantle/shopify/ShopifyServices.xml [hunk 3]
```diff
@@ -98,4 +99,27 @@
...
+            This service gets the location Id for the specific shop
+        </description>
+        <in-parameters>
+            <parameter name="systemMessageTypeId" required="true" />
+        </in-parameters>
+        <out-parameters>
+            <parameter name="locationId" />
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="moqui.service.message.SystemMessageRemote" value-field="systemRemoteIdentification">
```
Comment (marror7) [inline service/mantle/shopify/ShopifyServices.xml:114]: The name of the object on the value-field sentence could be SystemMessageRemote

**Review by rocha7778 — COMMENTED** [review]
I made the change

File: service/mantle/shopify/ShopifyServices.xml (hunk 3 above)
Comment (rocha7778) [reply service/mantle/shopify/ShopifyServices.xml]: this code is outdated

File: service/mantle/shopify/ApiServices.xml (hunk 1 above)
Comment (rocha7778) [reply service/mantle/shopify/ApiServices.xml]: the space was removed

**Pull request merged**